client/blocks/login/index.jsx

```diff
@@ -18,6 +20,11 @@ class Login extends Component {
 		twoFactorEnabled: PropTypes.bool,
 	};
 
+	state = {
+		hasSubmittedValidCredentials: false,
+		rememberMe: false,
+	};
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/13235#issuecomment-298366685
We might switch to a redux state in a later batch.